mcs/class/System/System.Media/SoundPlayer.cs

```diff
@@ -239,11 +239,12 @@ public void PlaySync ()
 				try {
 					if (adata == null)
 						adata = new WavData (mstream);
-					if (adev == null)
-						adev = AudioDevice.CreateDevice (null);
+					//if (adev == null)
```

Comment:
If this is not needed anymore then please remove it. If it's commented for a reason then leave a comment explaining why.